docs/getting_started.md

```diff
@@ -1,238 +1,204 @@
-learn more about how to create a variable group and
-[link](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=designer#use-a-variable-group) it to your pipeline.
-Click on **Library** in the **Pipelines** section as indicated below:
-
-![library_variable groups](./images/library_variable_groups.png)
-
-Create a variable group named **``devopsforai-aml-vg``**. The YAML pipeline definitions in this repository refer to this variable group by name.
+## Get the code
 
-The variable group should contain the following required variables:
-
-| Variable Name            | Suggested Value          |
-| ------------------------ | ------------------------ |
-| BASE_NAME                | [unique base name]       |
-| LOCATION                 | centralus                |
-| RESOURCE_GROUP           | mlops-RG                 |
-| WORKSPACE_NAME           | mlops-AML-WS             |
-| AZURE_RM_SVC_CONNECTION  | azure-resource-connection|
-| WORKSPACE_SVC_CONNECTION | aml-workspace-connection |
-| ACI_DEPLOYMENT_NAME      | diabetes-aci             |
-
-**Note:**
+If you intend to contribute back to this repository, fork the project. Otherwise use the [code template](https://github.com/microsoft/MLOpsPython/generate), which copies the entire code base to your own GitHub location with the git commit history restarted. You can use the resulting repository for this guide and for your own experimentation.
 
-The **WORKSPACE_NAME** parameter is used for the Azure Machine Learning Workspace creation. You can provide an existing AML Workspace here if you have one.
+## Create a Variable Group for your Pipeline
 
-The **BASE_NAME** parameter is used throughout the solution for naming
-Azure resources. When the solution is used in a shared subscription, there can
-be naming collisions with resources that require unique names like azure blob
-storage and registry DNS naming. Make sure to give a unique value to the
-BASE_NAME variable (e.g. MyUniqueML), so that the created resources will have
-unique names (e.g. MyUniqueMLamlcr, MyUniqueML-AML-KV, etc.). The length of
-the BASE_NAME value should not exceed 10 characters and it should contain numbers and letters only.
+You'll need to create a *variable group* in Azure DevOps to store values that are reused across multiple pipelines or pipeline stages. Either store the values directly in [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=designer#create-a-variable-group) or connect to an Azure Key Vault in your subscription. Check out the [Add & use variable groups](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml#use-a-variable-group) documentation to learn more about how to create a variable group and link it to your pipeline.
```

Comment:
Might be nice to have a second sentence which says "The example pipeline expects a variable group in order to set some key values" (or something like that) - basically, why am I doing this?

Reply:
Agreed. We also want to reference the section at the bottom with more info on the complete list of variables and how they can be toggled